examples/experimental/torch/classification/configs/mobilenet_v2_cifar100_asym_int8.json

```diff
@@ -0,0 +1,29 @@
+{
+    "model": "mobilenet_v2_32x32",
+    "pretrained": false,
+    "input_info": {
+      "sample_size": [2, 3, 32, 32]
+    },
+    "num_classes": 100,
+    "batch_size": 256,
+    "optimizer": {
+        "type": "Adam",
+        "base_lr": 1e-5,
+        "schedule_type": "multistep",
+        "steps": [
+            5
+        ]
+    },
+    "compression": {
+        "algorithm": "quantization",
+        "overflow_fix": "disable",
```

Comment:
This could be replaced by https://github.com/openvinotoolkit/nncf/blob/develop/examples/torch/classification/configs/quantization/mobilenet_v2_cifar100_asym_int8.json too.

Reply:
Please see my comment above.